### ISSUE TITLE
Bracketed citations in figure captions break highlighting

### ISSUE THREAD

**Issue by jhrmnn** [vim-pandoc/vim-pandoc-syntax#182]
![screen shot 2016-05-12 at 20 45 24](https://cloud.githubusercontent.com/assets/940353/15226330/8cc5da0a-1882-11e6-9dac-dc8094a0193e.png)

In fact, any brackets in the caption break the highlighting even though it's valid pandoc, but in those cases at least, one can escape them. Not so when they're part of a citation.

**Comment by rafaqz** [comment]
I'm experiencing the same problem in footnotes.

**Comment by holmescharles** [comment]
Ditto 4 years later. Is this still not fixed?

**Comment by alerque** [comment]
Apparently not. The entire synax set used by this plugin actually needs an overhaul. We've discussed some options, but nothing immanent.

We'd be happy to accept a PR that fixes this or other issues, but somebody needs to step up and contribute them. I personally don't use this syntax and am not likely to bump into it. I'd be happy to help facilitate a fix if anybody is going to wade in.

**Comment by holmescharles** [comment]
I'd help fix it, but I know next to nothing about how vim syntax highlighting works ☹️.

**Comment by alerque** [comment]
@holmescharles That's okay, it's kind of an esoteric field of knowledge. Unfortunately this plugin's syntax is probably not the one you want to get your feet wet with first, it's kind of a gargantuan mess!

**Comment by holmescharles** [comment]
Fair enough! This is really only an issue for a few of my figure captions
that reference other figures. So I will just ignore it. It's not a big deal.
Thanks!
